docs/core/install/macos.md

```diff
@@ -15,134 +19,251 @@ ms.custom: linux-related-content
+This article teaches you about which versions of .NET are supported on macOS, how to install .NET, and what the difference is between the SDK and runtime.
 
 The latest version of .NET is 8.
 
 > [!div class="button"]
 > [Download .NET](https://dotnet.microsoft.com/download/dotnet)
 
-## Supported releases
+## Supported versions
 
-There are two types of supported releases: Long Term Support (LTS) releases and Standard Term Support (STS) releases. The quality of all releases is the same. The only difference is the length of support. LTS releases get free support and patches for three years. STS releases get free support and patches for 18 months. For more information, see [.NET Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core).
+The following table lists the supported .NET releases, and which macOS they're supported on. These versions remain supported until either the version of [.NET reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of macOS is no longer supported.
 
-The following table is a list of currently supported .NET releases and the versions of macOS they're supported on:
+| macOS Version          | .NET     |
+|------------------------|----------|
+| macOS 15 "Sequoia"     | 8.0, 6.0 |
+| macOS 14 "Sonoma"      | 8.0, 6.0 |
+| macOS 13 "Ventura"     | 8.0, 6.0 |
 
-| Operating System       | .NET 8 (LTS) | .NET 6 (LTS) |
-|------------------------|--------------|--------------|
-| macOS 15 "Sequoia"     | ✔️ 8.0        | ✔️ 6.0        |
-| macOS 14 "Sonoma"      | ✔️ 8.0        | ✔️ 6.0        |
-| macOS 13 "Ventura"     | ✔️ 8.0        | ✔️ 6.0        |
-| macOS 12 "Monterey"    | ❌            | ❌            |
-| macOS 11 "Big Sur"     |              | ❌            |
-| macOS 10.15 "Catalina" |              | ❌            |
+[!INCLUDE [versions-not-supported](includes/versions-not-supported.md)]
 
-For a full list of .NET versions and their support life cycle, see [.NET Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core).
+## Runtime or SDK
 
-## Unsupported releases
+The **runtime** is used to run apps created with .NET. When an app author publishes an app, they can include the runtime with their app. If they don't include the runtime, it's up to the user to install the correct runtime.
 
-[!INCLUDE [versions-not-supported](includes/versions-not-supported.md)]
+There are two runtimes you can install on macOS, and both are included with the SDK.
 
-## Runtime information
+- _ASP.NET Core Runtime_\
+  Runs ASP.NET Core apps. Includes the .NET runtime. **Not available as an installer.**
 
-The runtime is used to run apps created with .NET. When an app author publishes an app, they can include the runtime with their app. If they don't include the runtime, it's up to the user to install the runtime.
+- _.NET Runtime_\
+  This runs standard console-based .NET apps.
```

Comment:
It might be worth saying that you can build ASP.NET web applications as a console app

Reply:
```suggestion
  This runs normal .NET apps, but not specialized apps, such as apps built on ASP.NET Core.
```